### PR TITLE
Add settings to monsterframes.lua

### PR DESCRIPTION
**Pull request by Grillo-del-mal**
Added a setting object to monsterframes.lua, just used the same implementation from expviewer :smile:

### CODE REVIEW

File: addons/monsterframes/monsterframes.lua [hunk 1]
```diff
@@ -62,20 +71,40 @@ function TGTINFO_TARGET_SET_HOOKED(frame, msg, argStr, argNum)
 		yPosition = 12;
 	end
 	
-	SHOW_PROPERTY_WINDOW(frame, monCls, targetinfo.raceType, "RaceType", xPosition + (0 * propertyWidth), yPosition, 10, 10);
-	SHOW_PROPERTY_WINDOW(frame, monCls, targetinfo.attribute, "Attribute", xPosition + (1 * propertyWidth), yPosition, 10, 10);
-	SHOW_PROPERTY_WINDOW(frame, monCls, targetinfo.armorType, "ArmorMaterial", xPosition + (2 * propertyWidth), yPosition, 10, 10);
-	SHOW_PROPERTY_WINDOW(frame, monCls, monCls["MoveType"], "MoveType", xPosition + (3 * propertyWidth), yPosition, 10, 10);
-	SHOW_PROPERTY_WINDOW(frame, monCls, nil, "EffectiveAtkType", xPosition + (4 * propertyWidth), yPosition, 10, 10);
+	local pos_idx = 0;
```
Comment (Excrulon) [inline addons/monsterframes/monsterframes.lua]: Mind renaming this to positionIndex so the variable names are spelled out and consistent in style?

File: addons/monsterframes/monsterframes.lua [hunk 2]
```diff
@@ -1,3 +1,12 @@
+settingsMonsterFrames = {
```
Comment (Excrulon) [inline addons/monsterframes/monsterframes.lua]: Please add a local to this so it's not on global scope.

```
local monsterFrameSettings = {
}
```

**Comment by Excrulon** [comment]
Made a few suggestions... oh and mind trying to resolve the merge conflicts?

**Comment by Grillo-del-mal** [comment]
done ;)

**Pull request merged**